src/6_branch.js

```diff
@@ -398,6 +398,12 @@ Branch.prototype['init'] = wrap(
 			}
 
 			var additionalMetadata = utils.getAdditionalMetadata();
+			var metadataFromInit = options && options['metadata'] && utils.validateParameterType(options['metadata'], "object") && Object.keys(options['metadata']).length > 0 ? options['metadata'] : null;
```

Comment:
options has already been defaulted to an empty object on line 302 so the options check is unnecessary. you can also remove options['metadata'] because that check will be handled in utils.validateParameterType